test_runner/src/main/kotlin/ftl/reports/xml/preprocesor/XmlPreprocessor.kt

```diff
@@ -0,0 +1,17 @@
+package ftl.reports.xml.preprocesor
+
+import org.apache.commons.text.StringEscapeUtils
+
+fun fixHtmlCodes(data: String): String {
```

Comment:
IMO expressions > blocks ;)
```
fun fixHtmlCodes(data: String): String = listOf(
    UtfControlCharRanges.CONTROL_TOP_START.charValue..UtfControlCharRanges.CONTROL_TOP_END.charValue,
    UtfControlCharRanges.CONTROL_BOTTOM_START.charValue..UtfControlCharRanges.CONTROL_BOTTOM_END.charValue
).flatten()
    .map { StringEscapeUtils.escapeXml11(it.toChar().toString()) }
    .filter { it.startsWith("&#") }
    .fold(data) { fixedStr: String, isoControlCode: String -> fixedStr.replace(isoControlCode, "") }
```
unions are not mandatory for those values

Reply:
I am also for @jan-gogo version 👍

Reply:
Thanks! Changed to better version :)